### PR TITLE
LPS-128719, LPS-128718, LPS-128716 | Master

### PR DESCRIPTION
**Pull request by Hanlf**
https://issues.liferay.com/browse/LPS-128719
https://issues.liferay.com/browse/LPS-128718
https://issues.liferay.com/browse/LPS-128716

### CODE REVIEW

**Comment by Hanlf** [comment]
ci:test:sf

**Comment by austinchiang** [comment]
**Reviewing...** :hourglass_flowing_sand: :sunglasses: :headphones:

File: portal-web/test/functional/com/liferay/portalweb/macros/BlogsEntry.macro [hunk 1]
```diff
@@ -361,16 +361,10 @@ definition {
...
-				AssertClick.assertPartialTextClickAt(
-					locator1 = "AssetPermissions#MORE_OPTIONS_LINK",
-					value1 = "More Options");
-			}
-
-			var key_permissionDefinitionKey = "${permissionDefinitionKey}";
-			var key_roleTitle = "${roleTitle}";
-
-			Check(locator1 = "AssetPermissions#PERMISSIONS_CHECKBOX");
+			PermissionsInline.addOrRemoveEntryPermissionsFromAllRoles(
```
Comment (austinchiang) [inline portal-web/test/functional/com/liferay/portalweb/macros/BlogsEntry.macro:364]: thx for the refactor

**Comment by austinchiang** [comment]
**Pull request submitted:** https://github.com/liferay-lima/liferay-portal/pull/1626<br></br>Please see changes [here.](https://github.com/austinchiang/liferay-portal/compare/5e1d7bfa8762~1...5e1d7bfa8762)

**Pull request closed without merging**